pkg/bundle/client.go

```diff
@@ -12,7 +12,7 @@ import (
 
 type Client interface {
```

Comment:
there is a general ticket to increase tests coverage for this module to 80%. I'm a bit reluctant to add tests now because that will only make the code move harder.

Reply:
Ok noted. I will keep an eye on https://github.com/aws/eks-anywhere-packages/issues/164.